hphp/parser/scanner.h

```diff
@@ -263,7 +263,7 @@ struct Scanner {
     incLoc(rawText, rawLeng, type);
   }
   void stepPos(const char *rawText, int rawLeng, int type = -1) {
-    if (shortTags()) {
+    if (full()) {
```

Comment:
I was very confused about the purpose of `stepPos()` until I realized that the reference to short tags was a bug introduced in https://github.com/facebook/hhvm/commit/7119bff22c524cfe66dfbd8ba5a9d1781381980f#diff-868580dfadd010b428015b3fc416a1acL247.